migration-engine/migration-engine-tests/tests/existing_data/mod.rs

```diff
@@ -779,7 +779,7 @@ fn set_default_current_timestamp_on_existing_column_works(api: TestApi) {
 }
 
 // exclude: there is a cockroach-specific test. It's unexecutable there.
-#[test_connector(preview_features("referentialIntegrity"), exclude(CockroachDb))]
+#[test_connector(preview_features("referentialIntegrity"), exclude(CockroachDb, TiDB))]
```

Comment:
Why does this not work on TiDB?

Reply:
Test result:
```
called `Result::unwrap()` on an `Err` value: ConnectorErrorImpl { user_facing_error: None, message: 
Some("Unsupported modify column: this column has primary key flag"), source: None, context: SpanTrace [] }
Unsupported modify column: this column has primary key flag
```

Reason:

> TiDB does not support adding or dropping clustered indexes after tables are created. Nor does it support the mutual conversion between clustered indexes and non-clustered indexes.

Doc: https://docs.pingcap.com/tidb/dev/clustered-indexes#add-or-drop-clustered-indexes